CONTRIBUTING.rst

```diff
@@ -0,0 +1,173 @@
+
+    ::
+
+        $ ENV\scripts\activate
+
+Note that if this works successfuly, your terminal prompt should look like:
+
+::
+
+    (ENV)$
+
+Finally, you should install vulture inside the activated virtualenv.
+
+In order to deactivate the ``virtualenv``, simply use
+
+::
+
+    (ENV)$ deactivate
+
+Installation
+------------
+
+There are two ways of installing vulture. By installing releases from PyPI, 
+or directly from source.
+
+PyPI Installation
+~~~~~~~~~~~~~~~~~
+
+::
+
+  $ pip install vulture
+
+Source installation
+~~~~~~~~~~~~~~~~~~~
+
+::
+
+  $ git clone https://github.com/jendrikseipp/vulture
+  $ cd vulture
+  $ pip install .
+
+Similarly, you can install your own fork of vulture.
+
+Testing
+-------
+
+Vulture uses tox for testing. You can read more about
+it at the `tox documentation <https://tox.readthedocs.io>`_.
+
+To install ``tox``, run:
+
+::
+
+  $ pip install tox
+
+To run ``tox``, simply use:
+
+::
+
+  $ tox
+
+Coding Standards
+----------------
+
+Creating a new branch
+~~~~~~~~~~~~~~~~~~~~~
+
+To start working on a pull request, you first need to create a new branch 
+where you will work. Do not change files when you are on your fork’s master 
+branch. The reason why you should never develop on your master branch is 
+because your fork’s master branch should always be synchronized with the 
+main repository’s master branch, which is much more challenging if it has 
+new commits on it. This is why we create our own branch:
+
+::
+
+  $ git checkout -b your-new-branch
+
+Naming the branches
+~~~~~~~~~~~~~~~~~~~
+
+A branch name should be descriptive about the feature/issue it is working on, 
+but at the same time, be a small string. Be careful not to re-use existing 
+branch names.
+
+Commits
+~~~~~~~
+
+Each commit should be atomic and only do the changes specified in it's messages.
+
+Commit messages are the way people in future, reading the code, can 
+understand changes. Make sure that they describe the changes in a clear and 
+crisp manner.
+
+Pull Request
+------------
+
+How to send a Pull Request?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To do this, you will have to go on GitHub, on your fork page. You should 
+change the branch to the one you have worked on and submitted the commit on. 
+Now you can create a Pull Request by clicking the ``New Pull Request`` button 
+in the pull request tab.
+
+Wait for reviews
+~~~~~~~~~~~~~~~~
+
+You now have to wait for the owners, and other contributors of the project to 
+review your changes.
+
+Follow-up
+~~~~~~~~~
+
+Now, there are two possibilities:
+
+- your PR will get accepted, and your commit will get merged into the master
+  branch
+
+- your PR will be rejected. There are 2 cases when a PR is rejected:
+
+    - Test fails
+    - Reviewer wants something changed
+
+In the case it is rejected, you should make the requested changes to your code 
+by making new commits, or amending existing commits using:
+
+::
+
+  $ git commit --amend
+
```

Comment:
Can we make this more concise, perhaps this:

In case your PR needs to be updated (tests fail or reviewer requests some changes), you may update the PR by either committing atop your branch or amending your previous commit (`git commit --amend`)